airflow/executors/__init__.py

```diff
@@ -57,31 +56,25 @@ def get_default_executor():
     return DEFAULT_EXECUTOR
 
 
-class Executors:
```

Comment:
I'm not immediately seeing why we need to pull this out of this file -- I think it was okay as it was?

(I'm not a huge fan of one-file-per class, espeicaly for small classes)

Reply:
The main reason is that we have some logic in the __init__. I wanted to make sure that wherever we are using import executors just to get the names of the executors we are not pulling in other dependencies. However after putting the imports to be local I might actually bring it back to __init__. I understand the concerns of a file per class and the micro-performance optimisation we can get.

Reply:
Brought it back.